src/BUILD

```diff
@@ -567,7 +567,7 @@ JAVA_TOOLS_DEPLOY_JARS = [
     "//conditions:default": [],
 })
 
-JAVA_VERSIONS = ("11",)
+JAVA_VERSIONS = ("11", "14")
 
 [
     genrule(
```

Comment:
Split this genrule into Java 11 and Java 14, and remove the (wrong version of) compilers from Java 14.

Reply:
I am not sure I follow. Are you suggesting not to use the variables, but copy/paste the entire toolchain genrule? But how that would work and scale for the next java toolchains: 15 and 17, ..., 42?

Reply:
Either no variables or you can add something like JAVA_VERSIONS_NO_JAVAC_JAR = ("14","15","17",...,"42") :)

Reply:
Done.